stories/Components/Select.stories.jsx

```diff
@@ -162,6 +162,27 @@ Grouped.args = {
   ],
 };
 
+const OptionTooltips = Template.bind({});
```

Comment:
Rename story to `OptionWithTooltip`.

```suggestion
const OptionWithTooltip = Template.bind({});
```